.github/workflows/ci.yml

```diff
@@ -250,6 +252,9 @@ jobs:
           B2_TOOLSET: ${{matrix.toolset}}
           B2_CXXSTD: ${{matrix.cxxstd}}
           B2_ADDRESS_MODEL: ${{matrix.addrmd}}
+          B2_CXXFLAGS: ${{ (matrix.toolset == 'msvc-14.3' && matrix.cxxstd == '14' && ( matrix.addrmd == '64' && '/RTCc' || '"/RTCc /arch:IA32"')) || '' }}
```

Comment:
You need to add `cxxflags` and `defines` to the matrix, instead of hardcoding checks here.